tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
 
-    "target": "ES2019",
+    "target": "ES5",
     "module": "commonjs",
     "lib": ["ES6", "DOM"],
```

Comment:
Why this change ? Why do you use `target ES5` and then `lib ES6` ?
For compatibility with old browsers but to be able to use `Promise` for example?

Reply:
@pasqualespica I found that setting target `ES5` and lib `ES6` supports `async`/`await` syntax for promises and also supports old browsers. I couldn't make things work with different choices